app/components/StackedProfiles.tsx

```diff
@@ -0,0 +1,51 @@
+import { tw, tx } from '@helpwave/common/twind/index'
+import type { Languages } from '@helpwave/common/hooks/useLanguage'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+
+type StackedProfilesTranslation = {
+  others: (users: number) => string
+}
+
+const defaultStackedProfilesTranslations: Record<Languages, StackedProfilesTranslation> = {
+  en: {
+    others: (users: number) => users === 1 ? 'other' : 'others'
+  },
+  de: {
+    others: (users: number) => users === 1 ? 'weiterer' : 'weitere'
+  }
+}
+
+// TODO replace later
+type UserDTO = {
```

Comment:
I think the color should not be a part of the user. We should inline these styles into `StackedProfilesProps` instead.

Reply:
There should be no color property on the user later on, only a link to profile picture as far as I know.

We should finalize the DataModels to make it easer on implementation

Reply:
Okay, can you then add a `TODO` to replace the colors with profile picture urls?

Reply:
Already updated it to use URL

Reply:
Currently there is no such thing as a "hover effect" with a tooltip for the user, this is something that could exist in the future though

But until then we don't need to talk about what the "user object" needs to look like, the avatar/profile picture component can just get a url and be done with it, no need to pass more than that to the avatar/profile picture component.